docs/modules/jenkins-shared-library/pages/orchestration-pipeline.adoc

```diff
@@ -248,6 +248,22 @@ As mentioned in the "Source Code Organisation" section, the orchestration pipeli
 def config = [debug: true, odsImageTag: 'x.x', versionedDevEnvs: true]
 ```
 
+=== Customizing the Release Manager configuration
+
+==== Timeouts
+
+If one of your components take longer than 10 minutes (this is the default value) to be promoted from one environment to another, the Release Manager pipeline will exit due to this timeout.
+You can increase this timeout by setting the `openshiftRolloutTimeoutMinutes` property in the `environments` section in the Release Manager repository in the `metadata.yml` file:
+
+```yaml
+...
+environments:
```

Comment:
See https://github.com/opendevstack/ods-jenkins-shared-library/blob/master/src/org/ods/orchestration/util/Project.groovy#L741.

`openshiftRolloutTimeoutMinutes` has to be configured per-environment, so underneath `prod`/`qa`.

Reply:
Yup.. same with retries